### PR TITLE
SEO-176062-xamarin-UG-issue

### PR DESCRIPTION
**Pull request by Edithabich**
Hi @MallikaRK ,
Have made changes on the redirect links. Please review.
Thanks.

Regards,
Edith.

### CODE REVIEW

File: Xamarin/Charts/Localization.md [hunk 1]
```diff
@@ -70,4 +70,4 @@ N> You can refer to our [Xamarin Charts](https://www.syncfusion.com/xamarin-ui-c
 
 ## See also
 
-[How to localize the labels in Xamarin.Forms Chart](https://www.syncfusion.com/kb/9415/how-to-localize-the-labels-in-xamarin-forms-chart)
\ No newline at end of file
+[How to localize the labels in Xamarin.Forms Chart](https://support.syncfusion.com/kb/article/8311/how-to-localize-the-labels-in-xamarin-forms-chart?isInternalRefresh=False)
```
Comment (Yvone-Atieno) [inline Xamarin/Charts/Localization.md:73]: Check this KB link

File: Xamarin/Charts/appearance.md [hunk 2]
```diff
@@ -369,4 +369,4 @@ N> You can refer to our [Xamarin Charts](https://www.syncfusion.com/xamarin-ui-c
 
 ## See also
 
-[How to apply custom fonts in Xamarin.Forms Chart](https://www.syncfusion.com/kb/9388/how-to-apply-custom-fonts-in-xamarin-forms-chart)
+[How to apply custom fonts in Xamarin.Forms Chart](https://support.syncfusion.com/kb/article/8209/how-to-apply-custom-fonts-in-xamarin-forms-chart?isInternalRefresh=False)
```
Comment (Yvone-Atieno) [inline Xamarin/Charts/appearance.md:372]: Check all KB links

**Review by Yvone-Atieno — CHANGES_REQUESTED** [review]
Commented

File: Xamarin/Charts/Localization.md (hunk 1 above)
Comment (Edithabich) [reply Xamarin/Charts/Localization.md]: resolved

File: Xamarin/Charts/appearance.md (hunk 2 above)
Comment (Edithabich) [reply Xamarin/Charts/appearance.md]: resolved

**Review by Yvone-Atieno — APPROVED** [review]
Approved

**Review by Aishwarya-Ganesan — APPROVED** [review]
fine,

**Pull request merged**